addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3804,13 +3813,19 @@ const ImageHandlerOption = SnippetOptionWidget.extend({
      * @see this.selectClass for parameters
      */
     setQuality(previewMode, widgetValue, params) {
+        if (previewMode) {
+            return;
+        }
```

Comment:
Instead of adding an explicit check for previewMode, you can add `data-no-preview="true"` in the we-input that calls the method, in which case it will stop receiving preview events entirely.

Reply:
The JS method that will be called does not handle the preview mode display, so I will have to add `data-no-preview = "true"` to all the widgets that call this method which will cause a duplication.

Reply:
Ok if indeed multiple widget use that method... however: is the preview mode not possible for those methods? Not efficient enough? If not, it's ok this way.